e2e/utils/index.ts

```diff
@@ -563,6 +564,11 @@ export function runCLI(
   }
 }
 
+export function readDependencyGraph(): ProjectGraph<any> {
```

Comment:
I don't think we need this function but if we do, then it should be called `readProjectGraph` or something like that.